modules/core/server/views/layout.server.view.html

```diff
@@ -17,7 +17,12 @@
     <meta name="fragment" content="!" />
     {% for cssFile in cssFiles %}
     <link rel="stylesheet" href="/{{cssFile}}?c={{appSettings.commit}}" />
-    {% endfor %} {% if env === 'development' %} {# in dev load js in head, as it
+    {% endfor %} {# Sentry DSN - make sure this is above scripts #} {% if
```

Comment:
I was about to comment that I'd just split these `endfor`+comment+`if` to their own lines for easier readability but realized the whole file is like this. 😁

Reply:
Yes, I think the prettier did it like that...